lnrpc/rpc.proto

```diff
@@ -1938,6 +1938,23 @@ message PendingChannelsResponse {
 
         /// The balance in satoshis encumbered in this channel
         int64 limbo_balance = 2;
+
+        /**
+        A list of valid commitment transactions. Any of these can confirm at
+        this point.
+        */
+        Commitments commitments = 3;
+    }
+
+    message Commitments {
```

Comment:
Is it necessary to have this all be named? We could just make this aa list of commit txs

Reply:
I was in doubt about that, because I use a plain list internally too. But I thought it may be useful for debugging to have them named.

Reply:
Yeah I can def see this being useful for debugging user tickets.

Reply:
Mhm, not that we will have to add many more fields here in the future.